pontoon/base/views.py

```diff
@@ -1064,9 +1064,8 @@ def amagama(request):
         return HttpResponse("error")
 
 
-def transvision(request, repo, title):
+def transvision(request):
     """Get Mozilla translations from Transvision service."""
-    log.debug("Get Mozilla translations from Transvision service.")
```

Comment:
Some of the machinery services require HTTP instead of HTTPS and also API keys (none of which is true for Transvision). And to make things consistent, I've moved it to view as well.